client-app/src/Components/DonatedItemsList.tsx

```diff
@@ -280,82 +153,11 @@ const DonatedItemsList: React.FC = () => {
                         >
                             <FaSearch />
                         </button>
-                    </div>
-
-                    <div className="dropdowns">
-                        <select className="sort-options" onChange={handleSort}>
-                            <option value="" disabled defaultValue="">
```

Comment:
We want all these sorting dropdowns. Original issue was to delete only few **searching** options and nowhere mentioned about the **sorting**. Please **revert** all the changes related to **sorting**